### DISCUSSION TITLE
criteria with a variable having no value or null

### DISCUSSION THREAD

**Discussion by Camts** [i3/i3#5736] [Q&A] [answered]
Hello,
I want to use criteria for `for_window` and `assign`.
For the same `class`, I have different cases:
* `title` contains a specific text (easy)
* `title` **is empty** (i have tried and not found how to)

My need is for SpringToolSuite:
* Having only editor in main window, placed on my main screen in workspace 2
* Having another window with views, placed on my side screen in workspace 10

Here is my example in `~/.config/i3/config` with `title=""` that does not work, my need is for SpringToolSuite.
```
for_window [class="SpringToolSuite4" title="Spring Tool Suite"] floating disable
assign [class="SpringToolSuite4" title="Spring Tool Suite"] 2
for_window [class="SpringToolSuite4" title=""] floating disable
assign [class="SpringToolSuite4" title=""] 10
```

**Comment by orestisfl** [comment, marked as answer]
Can't verify right now but can you try `title="^$"`

**Reply by Camts** [reply]
Thanks, it works.
`title` is absent when verified with `i3-msg -t get_tree`, but seems considered as empty since empty string regex works.
